BraveWallet/Settings/CustomNetworkDetailsView.swift

```diff
@@ -197,12 +212,45 @@ class CustomNetworkModel: ObservableObject, Identifiable {
     self.networkDecimals.input = String(network.decimals)
     if !network.rpcUrls.isEmpty {
       self.rpcUrls = network.rpcUrls.compactMap({ NetworkInputItem(input: $0) })
+    } else if mode.isViewMode {
+      self.rpcUrls = []
     }
     if !network.iconUrls.isEmpty {
       self.iconUrls = network.iconUrls.compactMap({ NetworkInputItem(input: $0) })
+    } else if mode.isViewMode {
+      self.iconUrls = []
     }
     if !network.blockExplorerUrls.isEmpty {
       self.blockUrls = network.blockExplorerUrls.compactMap({ NetworkInputItem(input: $0) })
+    } else if mode.isViewMode {
+      self.blockUrls = []
+    }
+  }
+}
+
+enum CustomNetworkError: LocalizedError, Identifiable {
+  case failed(errMsg: String)
+  case duplicateId
+
+  var id: String {
+    errorDescription
```

Comment:
May want to concatenate `errorTitle` with description to make it more unique

Reply:
Fixed in [e1981c1](https://github.com/brave/brave-ios/pull/5164/commits/e1981c1f3b8a7fae4b404a4dd260caa3d45dd1c4)